README.md

```diff
@@ -27,7 +80,7 @@ from fhir.resources.patient import Patient
 patients = Patient.fhir_bulk_import("patient_export.ndjson")
 ```
 
-### To FHIRflat
+#### To FHIRflat
```

Comment:
I have moved these to docs

Reply:
I've kept your changes and moved the CLI / raw data to FHIR stuff into a new page in the docs